docs/notebooks/failure_ego.pct.py

```diff
@@ -110,23 +110,11 @@ def create_classification_model(data):
 classification_model = create_classification_model(initial_data[FAILURE])
 
 # %% [markdown]
-# ## Create a custom optimize method
+# ## Build Trieste models
```

Comment:
it's too bad we won't show any more how to create a custom optimize method. OK for this PR, maybe just create a note somewhere?